src/Theme.js

```diff
@@ -2,6 +2,10 @@ export const theme = {
   color: {
     Light_grey: "#F5F5FA", // body background
     Black :"#18181B", // MovieList content's text
+    Mystic: "#E4E6F0",
+    Waterloo: "#7E839A",
+    PattensBlue: "#D6E4FF",
+    ScienceBlue: "#0044CC",
```

Comment:
Nice you used themes for colors. I would add comments for time being. They are always helpful when working in team :-)